.buildkite/scripts/bintray_publish.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/shared.sh
+
+if [[ "${FAKE_RELEASE_TAG}" ]]; then
+    echo "--- :warning: FAKE_RELEASE_TAG was found in the environment! This isn't a \"real\" release!"
+    fake_release=1
+fi
+
+publish(){
+    url=${1}
+    if [ -z "${fake_release}" ]; then
```

Comment:
Similar to above, either make this `${fake_release:-}` or make sure the variable is always defined.